### PR TITLE
Add time factor to MotionPlanRequest

### PR DESCRIPTION
**Pull request by skohlbr**
This is a change related to https://github.com/ros-planning/moveit_ros/pull/547 and adds a motion_execution_time_factor scaling factor for optionally slowing down trajectories.

### CODE REVIEW

File: msg/MotionPlanRequest.msg [hunk 1]
```diff
@@ -33,3 +33,14 @@ int32 num_planning_attempts
 
 # The maximum amount of time the motion planner is allowed to plan for (in seconds)
 float64 allowed_planning_time
+
+# The scaling factor for optionally scaling down the allowed maximum joint
+# velocities. The valid range is (0,1]. If a number outside this range is
+# specified, the 1.0 default value (i.e. max joint velocity) is used internally.
```
Comment (davetcoleman) [inline msg/MotionPlanRequest.msg]: It seems like you have the almost same comment twice. Remove this top one?

**Comment by davetcoleman** [comment]
Yea, I guess since there aren't likely any robots that use the current MotionPlanRequest.msg (and are stuck on some old version) its fine to change it in indigo. And we're still in beta anyway.

**Comment by mikeferguson** [comment]
Merging, even with double comment, since the moveit_core PR was merged (and devel build is now failing). I'll manually fix this in a moment

**Comment by davetcoleman** [comment]
Just a side note - the moveit_core PR didn't require this to be merged yet. Or did I miss something?

**Comment by mikeferguson** [comment]
yeah, I realized that after merging and releasing -- the PR failed to build after you merged it -- but I didn't catch in the log that it was actually a farm hiccup and not an actual build failure. We can still roll it back should we decide against putting this into indigo.

**Comment by davetcoleman** [comment]
I'm fine with it being merged, I see you fixed the comment.

**Comment by davetcoleman** [comment]
Another thing - do we really have to wait until next ROS release before merging the moveit_ros PR? I thought the buildfarm would source the new version anyway

**Comment by mikeferguson** [comment]
I think moveit_msgs needs to make it at least into shadow-fixed for the devel job to pass (only needs to be in shadow to build new debs properly)

**Comment by skohlbr** [comment]
Tanks for fixing the double comment and merging :)

**Pull request merged**